TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift

```diff
@@ -36,12 +36,12 @@ public protocol AuthenticatedSecureStorageProvider {
 ///:nodoc:
 public class AuthenticatedSecureStorage {
 
-  private enum Errors: Error, LocalizedError {
-    case authenticationFailed
+  public enum Errors: Error, LocalizedError {
+    case authenticationFailed(Error?)
 
-    var errorDescription: String? {
+    public var errorDescription: String? {
       switch self {
-        case .authenticationFailed: return "Authentication failed"
+      case .authenticationFailed: return "Authentication failed"
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/84517577f916f06073de7ffb2ba09f81b7655c5d/TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift#L44"></span>Case statements should be indented within their enclosing switch statement.<br />
<code>switch_case_alignment</code> <code>AuthenticatedSecureStorage.swift:44</code></td>
    </tr>
  </tbody>
</table>

Reply:
Could you check the indentation?

Reply:
Fixed, Thank you 👍